appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1818,105 +1911,95 @@ definitions:
       - status
       - _links
   PagedWeekActivityOverviews:
-    type: object
+    allOf:
+      - $ref: '#/definitions/Page'
     properties:
-      _links:
-        allOf:
-          - $ref: '#/definitions/Curies'
-          - $ref: '#/definitions/SelfLink'
-        properties:
-          next:
-            properties:
-              href:
-                type: string
-                description: Link to next page, if available
-            required:
-              - href
-          prev:
-            properties:
-              href:
-                type: string
-                description: Link to previous page, if available
-            required:
-              - href
-        required:
-          - next
-          - self
-          - prev
       _embedded:
         properties:
           "yona:weekActivityOverviews":
             type: array
             items:
               $ref: '#/definitions/WeekActivityOverview'
-      page:
-        description: Information about the current page
-        properties:
-          size:
-            type: string
-            description: The number of elements on the current page
-          totalElements:
-            type: string
-            description: The total number of elements
-            example: 300
-          totalPages:
-            type: string
-            description: The total number of pages
-            example: 15
-          number:
-            type: string
-            description: The current page number.
-            example: 1
+      _links:
+        allOf:
+          - $ref: '#/definitions/Curies'
+          - $ref: '#/definitions/SelfLink'
+          - $ref: '#/definitions/NextLink'
+          - $ref: '#/definitions/PrevLink'
+          - $ref: '#/definitions/FirstLink'
+          - $ref: '#/definitions/LastLink'
         required:
-          - size
-          - totalElements
-          - totalPages
-          - number
+          - self
     required:
       - _links
       - _embedded
       - page
   WeekActivityOverview:
     type: object
     properties:
-      _links:
-        allOf:
-          - $ref: '#/definitions/Curies'
-          - $ref: '#/definitions/SelfLink'
       date:
         type: string
         example: 2016-W2
         description: The week date in ISO 8601 format (YYYY-'W'w)
-      _embedded:
```

Comment:
You removed the embbedded resources with self link to retrieve more details because it returns different content I guess. That is an improvement, and I think maybe even required to have a consistent API.